lte/gateway/python/magma/enodebd/device_config/configuration_init.py

```diff
@@ -199,6 +199,8 @@ def _get_enb_config(
             tac = enb_config.tac
             bandwidth_mhz = enb_config.bandwidth_mhz
             cell_id = enb_config.cell_id
+            mme_address = enb_config.mme_ip
+            mme_port = 36412
```

Comment:
What is this value from? Are you adding configuration value for this as well?

Reply:
yes, I  use DEFAULT_S1_PORT and replace it.